index.html

```diff
@@ -9,6 +9,20 @@
   <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
   <link rel="manifest" href="/site.webmanifest">
   <title>Deal With It GIF emoji generator</title>
+  <meta property="og:type" content="website" />
+  <meta property="og:title" content="Deal With It GIF emoji generator" />
+  <meta property="og:description" content="All done artisanally and securely in your browser." />
+  <meta property="og:url" content="https://emoji.build/deal-with-it-generator/" />
+  <meta property="og:site_name" content="Deal With It GIF emoji generator" />
+  <meta property="og:image" content="https://emoji.build/deal-with-it-generator/favicon.ico" />
```

Comment:
```suggestion
  <meta property="og:image" content="/favicon.ico" />
```

There's favicons in https://github.com/klimeryk/dealwithit/tree/main/public - could you please use them instead? 🙏

Reply:
i.e., I should be using `https://emoji.build/favicon.ico` — is that right? I get a 404 on that URL:

```
$ wget https://emoji.build/favicon.ico
--2024-10-15 11:37:09--  https://emoji.build/favicon.ico
Resolving emoji.build (emoji.build)... 172.67.160.63, 104.21.9.152
Connecting to emoji.build (emoji.build)|172.67.160.63|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-10-15 11:37:09 ERROR 404: Not Found.
```

Reply:
No, just use `/favicon.ico`. Don't include the full domain URL. That will make it tied to that domain (which might not be what you want, for example, for local development or if deploying to a different domain for testing purposes). Normally, `/favicon.ico` would mean indeed mean `https://emoji.build/favicon.ico`. But Vite (the build system I'm using) automatically corrects the path at compile time: https://vite.dev/guide/build.html#public-base-path. So it ends up `https://emoji.build/deal-with-it-generator/favicon.ico`